packages/etherlime-lib/src/deployer/etherlime-ganache-deployer/etherlime-ganache-deployer.ts

```diff
@@ -13,46 +15,47 @@ class EtherlimeGanacheDeployer extends JSONRPCDeployer {
 	 * @param {*} port port number of the network to deploy on. This is the port number that is given to the class
 	 * @param {*} defaultOverrides [Optional] default deployment overrides
 	 */
-	constructor(privateKey = ganacheSetupConfig.accounts[0].secretKey, port = ganacheSetupConfig.defaultPort, defaultOverrides) {
+
+	constructor(privateKey: string = ganacheSetupConfig.accounts[0].secretKey, port: number = ganacheSetupConfig.defaultPort, defaultOverrides?: TxParams) {
 		EtherlimeGanacheDeployer._validatePortInput(port);
 
 		const nodeUrl = `http://localhost:${port}/`;
-
 		super(privateKey, nodeUrl, defaultOverrides);
 		this.nodeUrl = nodeUrl;
 	}
 
-	setPort(port) {
+	setPort(port: number): void {
 		EtherlimeGanacheDeployer._validatePortInput(port);
 		const nodeUrl = `http://localhost:${port}/`;
 		this.setNodeUrl(nodeUrl);
 	}
 
-	static _validatePortInput(port) {
+	static _validatePortInput(port: number): void {
 		if (!isNumber(port)) {
 			throw new Error(`Passed port (${port}) is not valid port`);
 		}
 	}
 
-	toString() {
+	toString(): string {
```

Comment:
For some class methods you've added access modifiers (public, private, etc). I think it is good idea to either keep putting them in all classes or none (I prefer the first)

Reply:
And just keep in mind that if we don't have a modifier, the default state is `public`.

Reply:
Yep, I've put modifiers only where the default should be changed. Shouldn't this method "toString()" be public?

Reply:
Most languages that have modifiers (like solidity) it is considered good practice to put explicitly the modifiers, just to form a habit of thinking of the access modification. It seems reasonable to me. What you guys think?

Reply:
The reason I omit them is the principe of "if anything can be skipped - skip it" and also I have read somewhere that not adding "public" distinguishes  "protected" and "private" and it's easy to see quickly what's what. But I could also agree with you!

Reply:
I agree with your point, lets skip them.